cmd/podman/machine/start.go

```diff
@@ -79,7 +83,23 @@ func start(_ *cobra.Command, args []string) error {
 	if err := vm.Start(vmName, startOpts); err != nil {
 		return err
 	}
+	if err := changeDockerContext(); err != nil {
+		fmt.Printf("Failed to change docker context to default. Error: %v", err)
+	}
 	fmt.Printf("Machine %q started successfully\n", vmName)
 	newMachineEvent(events.Start, events.Event{Name: vmName})
 	return nil
 }
+
+func changeDockerContext() error {
+	if runtime.GOOS != "darwin" {
+		return nil
+	}
+	path := os.Getenv("PATH")
+	path = strings.Join([]string{path, "/usr/bin", "/usr/local/bin"}, ":")
+	os.Setenv("PATH", path)
```

Comment:
Instead of changing the PATH globally for the whole process (which can cause side-effects on other areas of the code-base), a better approach would be to search a list of paths, which also has the added benefit to verify that docker binary exists before invoking it. Once found, the fully qualified joined absolute path could be used to invoke the specific location without needing a PATH change.

Reply:
I made the suggested change